packages/client/src/pages/LeaderBoard/LeaderBoard.tsx

```diff
@@ -1,21 +1,82 @@
-import { FC } from 'react'
+import { FC, useEffect, useState } from 'react'
 import Title from '@components/Title'
 import styles from './leaderBoard.module.css'
-import { playersData } from './stub'
 import TableRow from './elements/TableRow'
+import { teamName } from '@api/leaderBoard/constants'
+import { useDispatch, useSelector } from 'react-redux'
+import {
+    getLeaderboard,
+    leaderboardActions,
+    leaderboardSelectors,
+} from '@store/leaderboard'
+import { AppDispatch } from '@store'
+import { useAppSelector } from '@store/hooks'
 
-const LeaderBoard: FC = () => (
-    <div className={styles.container}>
-        <header className={styles.header}>
-            <Title className={styles.title}>Рейтинг</Title>
-            <hr className={styles.line} />
-        </header>
-        <div className={styles.table}>
-            {playersData.map((row, index) => (
-                <TableRow key={index} row={row} index={index} />
-            ))}
+const LeaderBoard: FC = () => {
+    const leaderboardData = useSelector(leaderboardSelectors.selectData)
```

Comment:
Здесь и ниже нужно использовать useAppSelector и useAppDispatch, они уже типизированы
```suggestion
    const leaderboardData = useAppSelector(leaderboardSelectors.selectData)
```